sql/hive/src/test/scala/org/apache/spark/sql/hive/security/HiveHadoopDelegationTokenManagerSuite.scala

```diff
@@ -52,7 +52,7 @@ class HiveHadoopDelegationTokenManagerSuite extends SparkFunSuite {
           throw new ClassNotFoundException(name)
         }
 
-        val prefixExcludeList = Seq("java", "scala", "com.sun.", "sun.")
+        val prefixExcludeList = Seq("java", "scala", "com.sun.", "sun.", "jdk.")
```

Comment:
Note: Log4j2 uses reflection in PatternParser constructor. On JDK11, it will cause `java.lang.IllegalAccessError` here when it accesses `jdk.internal.reflect.MagicAccessorImpl`.